BUILDING.md

```diff
@@ -94,6 +94,22 @@ apt-get install \
 	ctags
 ```
 
+#### Using Nix ####
+
+If you don't have [Nix](https://nixos.org/nix/), you can [get it](https://nixos.org/nix/download.html) with:
```

Comment:
We probably want to give a little more context here. In particular that:
 - This is an alternate way to install dependencies (the recommended/supported way is with apt)
 - This is focused on Mac support(?)

Reply:
I've added this to clarify: "Although the recommended platform is Debian or Ubuntu, Nix can be used to build VTR on other platforms."